channels/drive/client/drive_file.c

```diff
@@ -90,6 +90,33 @@ static BOOL drive_file_fix_path(WCHAR* path, size_t length)
 	return TRUE;
 }
 
+static BOOL contains_dotdot(const WCHAR* path, size_t base_length, size_t path_length)
```

Comment:
Thinking about - why don't we just let `_wcsstr` do the job i.e. `return _wcsstr(path, "/../") || _wcsstr(path, "\\..\\");`?

Reply:
we could, but there are 2 variants missing then ;)
(windows allows mixing `\\` and `/`)

Reply:
there is also the case when we start with `../` (ok, not in our specific case here as we don´t expect `base_length == 0`)

Reply:
True that - lgtm 👍